test/elixir_xml_to_map_test.exs

```diff
@@ -33,6 +33,10 @@ defmodule XmlToMapTest do
            }
   end
 
+  test "support for xmlns xml file" do
+    assert XmlToMap.naive_map(facebook_xmlns_xml(), [namespace_match_fn: &XmlToMap.default_namespace_match_fn/0]) == facebook_xmlns_xml_expected()
```

Comment:
you don't need to pass this in now right?  It's now the default match fn?

Reply:
Yep, the idea was to test the support for user predefined functions, but I forgot to set a function different from the default that we set! I will commit asap so we can merge :)

Reply:
Changed! I removed the unnecessary default function and included another test to show support for user's custom functions